buildstockbatch/postprocessing.py

```diff
@@ -628,8 +632,17 @@ def upload_file(filepath):
         bucket = s3.Bucket(s3_bucket)
         s3key = Path(s3_prefix_output).joinpath(filepath).as_posix()
         bucket.upload_file(str(full_path), str(s3key))
+    
+    def upload_buildstock_csv(filepath):
+        full_path = buildstock_dir.joinpath(filepath)
+        s3 = boto3.resource('s3')
+        bucket = s3.Bucket(s3_bucket)
+        s3_prefix_output_new = s3_prefix_output+ '/' + 'buildstock_csv' + '/'
+        s3key = Path(s3_prefix_output_new).joinpath(filepath).as_posix()
+        bucket.upload_file(str(full_path), str(s3key))
```

Comment:
When I look at the coverage report, this doesn't seem to be getting called. It's not clear to me why.

<img width="794" alt="image" src="https://github.com/NREL/buildstockbatch/assets/5325034/431ed64d-6f87-48e5-8f7b-625d926124e6">

It would make sense to add something to [this test](https://github.com/NREL/buildstockbatch/blob/348-upload-buildstockcsv-to-s3-during-postprocessing/buildstockbatch/test/test_base.py#L236) to check if the file got uploaded. It's using [mock](https://docs.python.org/3/library/unittest.mock.html) to not actually upload file to s3 but to see that it was called successfully.

Reply:
One more thought on this, could we not have `upload_buildstock_csv` here? It seems to mostly be a copy of the more generic `upload_file` but puts the file somewhere else. Maybe there could be a second, optional argument to `upload_file` where you could specify the s3 location when it's different than the default and then just use it for buildstock.csv.